pal/system-info/native/system-info.ts

```diff
@@ -88,7 +89,7 @@ class SystemInfo extends EventTarget {
 
         this.platform = platformMap[__getPlatform()];
         // eslint-disable-next-line max-len
-        this.isMobile = this.platform === Platform.ANDROID || this.platform === Platform.IOS || this.platform === Platform.OHOS || this.platform === Platform.OPENHARMONY;
+        this.isMobile = this.platform === Platform.ANDROID || this.platform === Platform.IOS || this.platform === Platform.OHOS || this.platform === Platform.OPENHARMONY || this.platform === Platform.QNX;
```

Comment:
QNX is not a mobile platform.

Reply:
'Platform.QNX' is a string, defined in pal/system-info/enum-type/platform.ts.

Reply:
Touch is not a criterion for judging mobile devices. Does it not work properly without being set as a mobile device?